Model/src/GamePackage/Player.java

```diff
@@ -1,4 +1,265 @@
+	 * 
+	 * @return propertiesOwned Returns the ArrayList propertiesOwned.
+	 *****************************************************************/
+	public ArrayList<BoardSquare> getPropertiesOwned() {
+		return propertiesOwned;
+	}
+
+	/******************************************************************
+	 * Setter method for propertiesOwned ArrayList.
+	 * 
+	 * @param propertiesOwned Sets the ArrayList propertiesOwned.
+	 *****************************************************************/
+	public void setPropertiesOwned(ArrayList<BoardSquare> 
+	propertiesOwned) {
+		this.propertiesOwned = propertiesOwned;
+	}
+
+	/******************************************************************
+	 * Getter method for cardsHeld ArrayList.
+	 * 
+	 * @return cardsHeld Returns the ArrayList cardsHeld.
+	 *****************************************************************/
+	public ArrayList<Card> getCardsHeld() {
+		return cardsHeld;
+	}
+
+	/******************************************************************
+	 * Setter method for cardsHeld ArrayList.
+	 * 
+	 * @param cardsHeld Sets the ArrayList cardsHeld.
+	 *****************************************************************/
+	public void setCardsHeld(ArrayList<Card> cardsHeld) {
+		this.cardsHeld = cardsHeld;
+	}
+	
+	/******************************************************************
+	 * Checks to make sure player has sufficient funds before
+	 * subtracting amount from wallet. If the player has sufficient
+	 * funds the method subtracts amount from wallet and returns the
+	 * amount paid. If player doesn't have sufficient funds he method
+	 * subtracts what is able to be paid from wallet and returns the
+	 * amount that has been paid.
+	 * 
+	 * @param amount The amount of money requested from player.
+	 * @return amountPaid The amount of money that was paid by player.
+	 *****************************************************************/
+	public int pay(int amount) {
+		int amountPaid = 0;
+		
+		if(wallet >= amount) {
+			wallet -= amount;
+			amountPaid = amount;
+		}
+		else {
+			amountPaid = amount - wallet;
+			wallet -= amountPaid;
+		}
+		
+		return amountPaid;
+	}
+	
+	/******************************************************************
+	 * Removes the property from the player's ArrayList of properties
+	 * owned.
+	 * 
+	 * @param property The property to be removed from owned properties
+	 * list.
+	 *****************************************************************/
+	public void sell(BoardSquare property) {
+		propertiesOwned.remove(property);
+	}
+	
+	/******************************************************************
+	 * Mortgages the player's property and adds the mortgage value to
+	 * their wallet.
+	 * 
+	 * @param property The property to be mortgaged.
+	 *****************************************************************/
+	public void mortgage(BoardSquare property) {
+		wallet += property.getMortgageValue();
+		property.setIsMortgaged(true);
+	}
+	
```

Comment:
mortgage() seems good, but I am thinking if we move sell() to Game, should we move this as well? Again, we need to have a group discussion about this. Probably not in the comments. (probably via slack or in person)

Reply:
Also, as it is, BoardSquare doesn't have a getMortgage() and a setIsMortGaged() methods, so this will not work unless we implement those methods on the BoardSquare class. a Solution would be for the railroad class to extend the PropertySquare class,  and for the parameter to be a PropertySquare.